### ISSUE TITLE
[ge] 对一个 torch npu 算子，单算子调用 vs GE 入图调用结果不一致

### ISSUE THREAD

**Issue by wwens7** [BD-Seed-HHW/xpu_graph#306]
### Describe the bug🐛

复现脚本：
```python
import torch
import torch_npu
import torchair as tng
from torchair.configs.compiler_config import CompilerConfig

def is_similar(a: torch.Tensor, b: torch.Tensor, rtol=1e-2, atol=1e-2):
    if a.shape != b.shape:
        print(f"Shape mismatch: {a.shape} vs {b.shape}")
        return False
    return torch.allclose(a.float(), b.float(), rtol=rtol, atol=atol)

device = torch.device("npu:0")
torch.npu.set_device(device)

config = CompilerConfig()
# config.mode = "reduce-overhead"
npu_backend = tng.get_npu_backend(compiler_config=config)

class NpuQuantMatmulModule(torch.nn.Module):
    def __init__(self):
        super().__init__()
        x2_data = torch.randint(0, 15, (16, 16), dtype=torch.int8, device=device)
        self.register_buffer("x2_data", x2_data)

    def forward(self, x1, scale):
        return torch.ops.npu.npu_quant_matmul.default(x1, self.x2_data, scale)

x1 = torch.randint(0, 15, (16, 16), dtype=torch.int8, device=device)
scale = torch.randn(16, dtype=torch.float32, device=device)
input_args = (x1, scale)

module_nd = NpuQuantMatmulModule().to(device)
result_direct_eager = module_nd(*input_args)
res_direct_cpu = result_direct_eager.cpu()

compiled_manual_module = torch.compile(module_nd, backend=npu_backend, dynamic=False)
result_compiled_manual = compiled_manual_module(*input_args)
res_compiled_manual_cpu = result_compiled_manual.cpu()

are_similar_ac = is_similar(res_direct_cpu, res_compiled_manual_cpu)

if are_similar_ac:
    print("  ✅ PASS: Compiler correctly handles pre-optimized graph.")
else:
    print("  ❌ FAIL: Compiler failed to handle pre-optimized graph correctly!")
```

### Environment details

torch                  2.5.1+cpu
torch-npu              2.5.1
opp_kernel_running_version=[7.7.0.1.217:8.0.T112]
runtime_running_version=[7.7.0.1.217:8.0.T112]
compiler_running_version=[7.7.0.1.217:8.0.T112]
hccl_running_version=[7.7.0.1.217:8.0.T112]
opp_running_version=[7.7.0.1.217:8.0.T112]
toolkit_running_version=[7.7.0.1.217:8.0.T112]
aoe_running_version=[7.7.0.1.217:8.0.T112]
ncs_running_version=[7.7.0.1.217:8.0.T112]
runtime_upgrade_version=[7.7.0.1.217:8.0.T112]
compiler_upgrade_version=[7.7.0.1.217:8.0.T112]
hccl_upgrade_version=[7.7.0.1.217:8.0.T112]
opp_upgrade_version=[7.7.0.1.217:8.0.T112]
toolkit_upgrade_version=[7.7.0.1.217:8.0.T112]
aoe_upgrade_version=[7.7.0.1.217:8.0.T112]
ncs_upgrade_version=[7.7.0.1.217:8.0.T112]
opp_kernel_upgrade_version=[7.7.0.1.217:8.0.T112]
runtime_installed_version=[7.6.T11.0.B080:8.0.0.alpha003][7.7.0.1.217:8.0.T112]
compiler_installed_version=[7.6.T11.0.B080:8.0.0.alpha003][7.7.0.1.217:8.0.T112]
hccl_installed_version=[7.6.T11.0.B080:8.0.0.alpha003][7.7.0.1.217:8.0.T112]
opp_installed_version=[7.6.T11.0.B080:8.0.0.alpha003][7.7.0.1.217:8.0.T112]
toolkit_installed_version=[7.6.T11.0.B080:8.0.0.alpha003][7.7.0.1.217:8.0.T112]
aoe_installed_version=[7.6.T11.0.B080:8.0.0.alpha003][7.7.0.1.217:8.0.T112]
ncs_installed_version=[7.6.T11.0.B080:8.0.0.alpha003][7.7.0.1.217:8.0.T112]
opp_kernel_installed_version=[7.6.T11.0.B080:8.0.0.alpha003][7.7.0.1.217:8.0.T112]
